### DISCUSSION TITLE
Add option to change the language of the weekdays or to the add event menu and more

### DISCUSSION THREAD

**Discussion by Huddeij** [davish/obsidian-full-calendar#431] [Feature Requests]
I'd love to be able to localize this plugin to my dates, language and more, like the rest of Obsidian

**Comment by davish** [comment]
This is related to #72. Localization is a notoriously hard problem for web apps, and since this is a project in my spare time, I'm not sure if I can take it on. Especially as someone who only speaks 1.5 languages. I will tag these issues as "help wanted", in case others can help contribute on this front.

**Comment by supersmart-at** [comment]
hi davish, 
just found this settings in fullcalender API: https://fullcalendar.io/docs/locale
If i understand @Trigant right, that should meet his requirements. 
- For me as native german speaker it is ok to have an additional plugin setting to initialize the fullcalendar in my language. 

Here is the demo language switcher: https://fullcalendar.io/docs/locale-demo
